lib/internal/util/inspect.js

```diff
@@ -308,8 +308,7 @@ function inspect(value, opts) {
     if (typeof opts === 'boolean') {
       ctx.showHidden = opts;
     } else if (opts) {
-      const optKeys = ObjectKeys(opts);
-      for (const key of optKeys) {
+      ArrayPrototypeForEach(ObjectKeys(opts), (key) => {
```

Comment:
```suggestion
      const optKeys = ObjectKeys(opts);
      for (let i = 0; i < optKeys.length; ++i) {
        const key = optKeys[i];
```
@aduh95 I was considering this change because this prevents the construction of the lambda function object, every time `inspect` is called.